root/statistics/LanguagesScripts.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2018 Shamroy Pellew
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+const React = require('react');
+const manifest = require('../static/manifest');
+const {l} = require('../static/scripts/common/i18n');
+const Layout = require('./Layout');
+
+const formatCount = (n) => {
+  return n.toLocaleString($c.stash.current_language);
+};
+
+const LinkSearchableProperty = ({
+  searchField,
+  searchValue,
+  entityType,
+  text,
+}) => {
+  const params = {limit: 25, method: 'advanced', query: searchField + ':\"' + searchValue + '\"', type: entityType};
+  return (
+    <a href={'/search?' + Object.keys(params).map(key => key + '=' + params[key]).join('&')}>
```

Comment:
You're repeating this pattern in every file, so it'd be better to move it into a utility function and import it.

The values also need to be escaped properly using `encodeURIComponent`.

Reply:
Actually, you can use `$.param`: http://api.jquery.com/jquery.param/